inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/iceberg/src/main/java/org/apache/inlong/sort/iceberg/sink/multiple/IcebergSingleStreamWriter.java

```diff
@@ -94,6 +105,8 @@ public IcebergSingleStreamWriter(
         this.dirtyOptions = dirtyOptions;
         this.dirtySink = dirtySink;
         this.multipleSink = multipleSink;
+        this.tableSchema = tableSchema;
+        this.cachedWriteResults = new LinkedList<>();
```

Comment:
Maybe use ArrayList is better. You just append add element to tail

Reply:
Fixed